### PR TITLE
qa: change mon_bind_msgr2 to ms_bind_msgr2

### PR DESCRIPTION
**Pull request by smithfarm**
This brings the ceph task into line with the state of the ceph.conf option
("ms bind msgr2" instead of "mon bind msgr2")

Fixes: https://tracker.ceph.com/issues/38216

Signed-off-by: Nathan Cutler <ncutler@suse.com>

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by smithfarm** [comment]
@yuriw Could you check if this really fixes https://tracker.ceph.com/issues/38216 ?

File: qa/tasks/ceph.py [hunk 1]
```diff
@@ -653,7 +653,7 @@ def cluster(ctx, config):
            (remote.ssh.get_transport().getpeername() for (remote, role_list) in remotes_and_roles)]
     mons = get_mons(
         roles, ips, cluster_name,
-        mon_bind_msgr2=config.get('mon_bind_msgr2'),
+        ms_bind_msgr2=config.get('ms_bind_msgr2'),
         mon_bind_addrvec=config.get('mon_bind_addrvec'),
```
Comment (smithfarm) [inline qa/tasks/ceph.py]: There does not seem to be any ceph.conf option called `mon_bind_addrvec` (or even `ms_bind_addrvec` for that matter) so I wonder if setting `mon_bind_addrvec` actually does anything?

**Comment by liewegas** [comment]
I made these different on purpose to avoid confusion between them and config options.  They control whether the cluster's monmap gets a v2 address and whether it gets multiple addrs (addrvec).

**Comment by smithfarm** [comment]
@liewegas Thanks for clarifying. I get it, now!

**Pull request closed without merging**